bentoml/pytorch.py

```diff
@@ -230,28 +232,33 @@ def _setup(self) -> None:  # type: ignore[override]
     @torch.no_grad()
     def _run_batch(  # type: ignore[override]
```

Comment:
I don't think we need this `# type: ignore` anymore
```suggestion
    def _run_batch(
```

Reply:
let me correct this part for all frameworks in another commit